config/settings.yml

```diff
@@ -53,6 +53,8 @@ api_timeout: 300
 web_timeout: 30
 # Periods (in hours) which are allowed for user blocks
 user_block_periods: [0, 1, 3, 6, 12, 24, 48, 96, 168, 336, 731, 4383, 8766, 87660]
+# Account deletion cooldown period (in hours) since last changeset close; -24 to disable, 0 to make sure there aren't any open changesets when the deletion happens
+user_account_deletion_delay: -24
```

Comment:
I'm not sure what the answer is but a negative cooldown period is clearly a nonsensical idea so we really need to find some better way of expressing this...

Reply:
It's as nonsensical as a future close date.

It's possible to compare the current time with min(close time, current time), but then we throw away the check for open changesets.

Reply:
I can add null as a 'disable' value to settings.yml.

Reply:
I'm working on a solution, I should have something to share shortly.

Reply:
https://github.com/gravitystorm/openstreetmap-website/commit/7802efc8c986bf0648dccd8f1f80cca631cf3420 - I think the clearest thing is to have two configuration options - one to turn cooldown on and off, and the other to configure the length of the cooldown.

What do you both think? I prefer the separate boolean config, rather than having one option with mixed integer/nil (where 0 and nil will have significantly different behaviours).

Reply:
I already did null.

I don't care if it's null or extra boolean option or anything else.

Reply:
OK, let's go for that then.